### PR TITLE
Documenting the tags added to cucushift

### PR DESCRIPTION
**Pull request by jhou1**
This documents the tags used in cucushift, will be used as a reference for contributors to follow. 
@liangxia @pruan-rht @JianLi-RH @dis016 PTAL

### CODE REVIEW

**Comment by jhou1** [comment]
Resolves https://issues.redhat.com/browse/OCPQE-5528

File: doc/tags.adoc [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+|`@destructive`
+|Test scenarios that perform operations which would result in failures of other parallel tests or create an outage/downtime to the cluster. Scenarios including: node draining, node restart, scaling down, etc.
+
+|`@inactive`
+|Inactive test scenarios, either outdated or no longer applicable. Skip tests with an `@inactive` tag
+
+|`@flaky`
+|Unreliable test that keeps failing and is recognized as test automation bug. Skip tests with a `@flaky` tag
+
+|`@prow_unstable`
```
Comment (liangxia) [inline doc/tags.adoc:24]: perhaps `prow_flaky` to be consistent ?

File: doc/tags.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+|Usage Scenario
+
+|`@admin`
+|Test scenarios that perform operations requiring the `cluster-admin` priviledge
+
+|`@destructive`
+|Test scenarios that perform operations which would result in failures of other parallel tests or create an outage/downtime to the cluster. Scenarios including: node draining, node restart, scaling down, etc.
+
+|`@inactive`
+|Inactive test scenarios, either outdated or no longer applicable. Skip tests with an `@inactive` tag
```
Comment (pruan-rht) [inline doc/tags.adoc:19]: Inactive test scenarios that are either...

File: doc/tags.adoc (hunk 1 above)
Comment (jhou1) [reply doc/tags.adoc:24]: Initially, I only wanted the `@flaky` tag. I realize we should give the team more time to triage their unstable tests in Prow, so used a different name to distinguish them.

**Review by JianLi-RH — APPROVED** [review]

**Pull request merged**